connectors/cosmos/connector.go

```diff
@@ -58,47 +58,57 @@ type Connector struct {
 type ConnectorSettings struct {
 	ConnectionString string
 
-	serverConnectTimeout           time.Duration
-	pingTimeout                    time.Duration
+	ServerConnectTimeout           time.Duration
+	PingTimeout                    time.Duration
 	InitialSyncNumParallelCopiers  int
-	writerMaxBatchSize             int //0 means no limit (in # of documents)
+	WriterMaxBatchSize             int //0 means no limit (in # of documents)
 	NumParallelWriters             int
 	CdcResumeTokenUpdateInterval   time.Duration
-	numParallelIntegrityCheckTasks int
+	NumParallelIntegrityCheckTasks int
 
-	maxNumNamespaces            int    //we don't want to have too many parallel changestreams (after 10-15 we saw perf impact)
-	targetDocCountPerPartition  int64  //target number of documents per partition (256k docs is 256MB with 1KB average doc size)
-	numParallelPartitionWorkers int    //number of workers used for partitioning
+	MaxNumNamespaces            int    //we don't want to have too many parallel changestreams (after 10-15 we saw perf impact)
+	TargetDocCountPerPartition  int64  //target number of documents per partition (256k docs is 256MB with 1KB average doc size)
+	NumParallelPartitionWorkers int    //number of workers used for partitioning
 	partitionKey                string //partition key to use for collections
 
 	EmulateDeletes         bool // if true, we will generate delete events
-	deletesCheckInterval   time.Duration
+	DeletesCheckInterval   time.Duration
 	WitnessMongoConnString string
 }
 
 func NewCosmosConnector(desc string, settings ConnectorSettings) *Connector {
 	// Set default values
-	settings.serverConnectTimeout = 15 * time.Second
-	settings.pingTimeout = 2 * time.Second
-
-	if settings.InitialSyncNumParallelCopiers == 0 { //default to 8
+	if settings.ServerConnectTimeout == 0 { // default 15 seconds
```

Comment:
This if xxx == 0 {...} business is starting to look ugly. Can we use something like this instead? https://pkg.go.dev/github.com/carlmjohnson/truthy#First